ext/nnstreamer/tensor_filter/tensor_filter_llamacpp.cc

```diff
@@ -251,14 +249,24 @@ TensorFilterLlamaCpp::configure_instance (const GstTensorFilterProperties *prop)
 
   ctx = llama_init_from_model (model, ctx_params);
   if (ctx == nullptr) {
+    cleanupResources ();
     throw std::runtime_error ("Failed to create llama context");
   }
 
+  if (lora_adapter) {
+    llama_set_adapter_lora (ctx, lora_adapter, 1.0f);
```

Comment:
minor comment, I suggest removing lora_path, and lora_adapter if possible.
```
if (prop->num_models == 2) {
  llama_adapter_lora *lora_adapter = llama_adapter_lora_init (prop->model_files[1]);
  if (llama_set_adapter_lora (ctx, lora_adapter, 1.0f) != 0) {
    // handle error
  }
}
```

Reply:
Finally, I fixed the code. Thank you.